tests/testthat/test_downloadable_react_table.R

```diff
@@ -0,0 +1,22 @@
+context("periscope2 - downloadable react table")
+local_edition(3)
```

Comment:
I was wondering why we had to set the local_edition. Would it be helpful to add comment to this so we know why this was done?

Reply:
ah ok thanks!